tests/lwip_sock_udp/Makefile

```diff
@@ -24,10 +24,12 @@ USEMODULE += netdev_eth
 USEMODULE += netdev_test
 USEMODULE += ps
 USEMODULE += sock_udp
+USEMODULE += sock_aux_local
```

Comment:
Same modeling for `AUX_LOCAL` should be applied here, if suggestion above is accepted

Reply:
Done